docs/csharp/whats-new/tutorials/snippets/staticinterfaces/Program.cs

```diff
@@ -9,6 +9,11 @@
     Console.WriteLine(str++);
 // </TestRepeat>
 
+// <TestUniqueInstance>
+var instance = UniqueInstance.Instance;
```

Comment:
For this to work, you'll need to use the interface type:

```suggestion
var instance = (UniqueInstance as ISingleton)?.Instance;
```

Reply:
@BillWagner I committed this suggestion, although locally, with VS still on version 17.4 I do have issues with your suggestion: CS0119 for UniqueInstance and CS0305 for ISingleton (which can be fixed by specifying the T type for this cast).
Let's see if remotely it will also fail and then we can check.

Reply:
@BillWagner I should mention, that locally, with VS v17.4 I can also see the initial error for `Instance` not being a part of `UniqueInstance`.
Somewhere I read that `static virtual` feature requires:
* .NET v8.0 or above - locally I'm running on .NET 9, and I see this `staticinterfaces` project runs on .NET 8, which should be fine?
* preview enabled as `LangVersion` - we have that.
* VS 17.6 or above - this is what I'm missing locally, but can't say regarding the CI checks.